### PR TITLE
Errors that might occur during the presentation

### PR DESCRIPTION
**Pull request by peppelinux**
this PR aims to close https://github.com/italia/eudi-wallet-it-docs/issues/224

since we still don't have the required details in the openid4vp specs, I propose a relaxed language and generic error identifiers.

I hope that this little step might help the discussion within the openid working group.

### CODE REVIEW

**Comment by peppelinux** [comment]
See also
https://github.com/openid/OpenID4VP/issues/120

File: docs/en/remote-flow.rst [hunk 1]
```diff
@@ -373,6 +373,49 @@ The JWS payload parameters are described herein:
   - ``client_metadata_uri``: string containing an HTTPS URL pointing to a resource where a JSON object with the Relying Party metadata can be retrieved. The ``client_metadata_uri`` parameter MUST NOT be present when ``client_id_scheme`` is ``entity_id``. Since the ``client_metadata`` is taken from ``trust_chain``, this parameter is intended to not be used.
 
 
+Request URI Endpoint Errors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When the Reying Party encounters errors while issuing the Request Object from the ``request_uri`` endpoint, the following error responses are applicable:
```
Comment (giadas) [inline docs/en/remote-flow.rst]:
```suggestion
When the Relying Party encounters errors while issuing the Request Object from the ``request_uri`` endpoint, the following error responses are applicable:
```

File: docs/en/remote-flow.rst [hunk 2]
```diff
@@ -503,6 +580,16 @@ The following is a non-normative example of the response from the Relying Party
...
 
+Redirect URI Errors
+-------------------
+
+When the Wallet Instance sends the user-agent to the Redirect URI provided by the Relying Party, several errors may occur that prevent the successful completion of the process. These errors are critical as they directly impact the User experience by hindering the seamless flow of information between the Wallet Instance and the Relying Party. Below are potential errors related to the Redirect URI and their implications:
+
+- **Mismatched Redirect URI**: This error occurs when the Redirect URI provided by the Relying Party does not match any of the URIs linked with the User session. This mismatch can lead to a HTTP status error code set to 403 (Forbidden), indicating that the request cannot be processed due session/URI mismatch.
+
+- **Redirect URI Security Issues**: If the Relying Party incours in security issues when evaluating te User session with the provided URI, the Relying Party MUST raise an error. In such cases, an HTTP status code set to 403 (Forbidden) MUST be returned, indicating that the request is valid but the server is refusing action due to security precautions.
```
Comment (giadas) [inline docs/en/remote-flow.rst]:
```suggestion
- **Redirect URI Security Issues**: If the Relying Party incurs in security issues when evaluating the User session with the provided URI, the Relying Party MUST raise an error. In such cases, an HTTP status code set to 403 (Forbidden) MUST be returned, indicating that the request is valid but the server is refusing action due to security precautions.
```

**Review by fmarino-ipzs — APPROVED** [review]
I just fixed a couple of editorial issues (the figure was not displayed correctly in the document and a dot was missing in the warning section resulting in a block comment of the text below).

**Pull request merged**